_includes/footer.html

```diff
@@ -1,4 +1,4 @@
-<div style="background-color: #F1F7F8;">
+<div class="bg-gray-100">
```

Comment:
we should now replace the definition of gray 100 in 

https://github.com/coopdevs/coopdevs.github.io/blob/20a38b214ffc201f17ba7f00f79ec16dfeacfad5/_includes/tailwind.config.js#L6

with the color we designed, `#F1F7F8`